DependencyInjection/Configuration.php

```diff
@@ -34,8 +34,7 @@ public function getConfigTreeBuilder()
                     return !empty($v['classes']['client'])
                         || !empty($v['classes']['message_factory'])
                         || !empty($v['classes']['uri_factory'])
-                        || !empty($v['classes']['stream_factory'])
-                    ;
+                        || !empty($v['classes']['stream_factory']);
```

Comment:
is this styleci? my idea was that adding another check only does a diff on one line, like trailing comma in arrays.